lib/logfmt/encoder.ex

```diff
@@ -30,6 +30,11 @@ defmodule Logfmt.Encoder do
   end
 
   @spec encode_value(value :: term) :: String.t()
+  defp encode_value(value)
+
+  defp encode_value(""),
+    do: ~s("")
```

Comment:
Does it make sense to put this in the [BitString encoder](https://github.com/jclem/logfmt-elixir/blob/81a33edea9f4f5b84b7a50c4b9da2835ec8bba6b/lib/logfmt/value_encoder.ex#L6-L8), instead?

Reply:
The bitstring encoder just encodes the string, but it doesn't quote it.
It works like an identity function for strings, `encode_value` adds the surrounding `"`, which is the function that I'm adding that special case for empty strings.